web/src/components/Plugin/locales/zh-CN.ts

```diff
@@ -21,6 +21,16 @@ export default {
   'component.step.select.pluginTemplate.select.option': '手动配置',
   'component.plugin.pluginTemplate.tip1': '1. 若路由已配置插件，则插件模板数据将与已配置的插件数据合并。',
   'component.plugin.pluginTemplate.tip2': '2. 插件模板相同的插件会覆盖掉原有的插件。',
+  'component.plugin.general': '通用',
```

Comment:
Those categories are referring to Konga and  https://github.com/apache/apisix-dashboard/blob/053721cc483c6ffc21981b1d6a70cfce07aa808b/web/src/components/Plugin/data.tsx#L21